index.html

```diff
@@ -9,12 +9,18 @@
 <body>
   <header class="main-header">
     <nav class="site-navigation">
-      <p class="main-logo">Техномарт</p>
+      <p class="main-logo"><img src="img/svg/technomart.svg" alt="Техномарт"></p>
       <ul>
         <li>
-          <!-- Здесь будет форма -->Поиск</li>
-        <li><button type="button">Закладки</button></li>
-        <li><button type="button">Корзина</button></li>
+          <!-- Здесь будет форма -->
+          < src="img/svg/search.svg" alt="Поиск">Поиск
```

Comment:
```suggestion
          Поиск
```
Это декторативная картинка, поэтому её не нужно вставлять как контентную, через тег `<img>`